spdx/parsers/tagvaluebuilders.py

```diff
@@ -955,6 +960,78 @@ def set_pkg_comment(self, doc, text):
         else:
             raise CardinalityError("Package::Comment")
 
+    def set_pkg_primary_package_purpose(self, doc, purpose):
+        """
+        Set the package's primary purpose.
+        Raise CardinalityError if more than one purpose is set.
+        Raise SPDXValueError if purpose is unknown.
+        """
+        self.assert_package_exists()
+        if not self.package_primary_purpose_set:
+            self.package_primary_purpose_set = True
+            for purpose_enum in PackagePurpose:
+                if purpose == purpose_enum.name:
+                    doc.packages[-1].primary_package_purpose = purpose_enum
+                    return True
```

Comment:
Not sure if the returned boolean serves any value, but it seems to be the pattern here :man_shrugging:

Reply:
there are several analog methods that do not have this `return True`, so I suppose it is unnecessary, but I haven't checked the whole code yet

Reply:
Yeah, let's not spend too much time analyzing these builders now